pyteal/ast/abi/util.py

```diff
@@ -1,4 +1,14 @@
-from typing import Sequence, TypeVar, Any, Literal, get_origin, get_args, cast
+from typing import (
+    Sequence,
+    TypeVar,
+    Any,
+    Literal,
+    List,
+    Union,
+    get_origin,
+    get_args,
+    cast,
```

Comment:
```suggestion
    Any,
    List,
    Literal,
    Sequence,
    TypeVar,
    Union,
    cast,
    get_args,
    get_origin,
```

nit: ocd request to sort the imports

Reply:
you may not actually need all of these after converting the `typing` types to concrete types as suggested below and which conforms with 3.10 usage.